posthog/hogql/database/schema/cohort_people.py

```diff
@@ -67,7 +67,7 @@ def to_printed_hogql(self):
 class CohortPeople(LazyTable):
     fields: Dict[str, FieldOrTable] = COHORT_PEOPLE_FIELDS
 
-    def lazy_select(self, requested_fields: Dict[str, List[str | int]], modifiers: HogQLQueryModifiers):
+    def lazy_select(self, requested_fields: Dict[str, List[str | int]], modifiers: HogQLQueryModifiers, node):
```

Comment:
Very much a nit, but could we add the type of `node` to all of these signatures please

Reply:
Why are they needed? Looks like they are inferred correctly from the superclass

Reply:
They're not needed whatsoever I think. But all the other parameters have typed and it's not obvious that the func is an override of the base class. Again, very much a nit and so feel free to not do it 😄